charts/observability-pipelines-worker/values.yaml

```diff
@@ -32,10 +32,11 @@ datadog:
   dataDir: "/var/lib/observability-pipelines-worker"
   workerAPI:
     # datadog.workerAPI.enabled -- Whether to enable the Worker's API.
-    enabled: false
+    enabled: true
     # datadog.workerAPI.playground -- Whether to enable the Worker's API GraphQL playground.
-    playground: true
+    playground: false
     # datadog.workerAPI.address -- Local address to bind the Worker's API to.
+    # if you change this port, you'll need to update the livenessProbe and readinessProbe
     address: "127.0.0.1:8686"
```

Comment:
@sghall was actually playing with k8s for another customer and went to enable liveness probe, and I noted this actually needs to be `0.0.0.0` -- I'd done this previously here: https://github.com/DataDog/logs-psa-private/blob/1776a2b0028ab28058986c568fdf5b21bbb6c7c0/product-testing/OP-Google-Archive/values-w-volume-mount.yaml#L9-L10 -- 127.0.0.1 will only listen on the local interface. So we will probably want to update this PR to reflect that. I'll do that now.

Reply:
@ckelner ack.  Could you also clone this repo and PR this using a regular branch?  I'm not saying it won't work but it's a bit unusual to merge it from a fork of the repo.  I can't tell if that's causing some of the CI failures here.

Reply:
I was trying to resolve the CI issues over here https://github.com/DataDog/helm-charts/pull/1997

Reply:
I didn't have writes to the repo which is why I created a branch. When I try to push a new branch I get the following:

```
git push origin ckelner/op-k8s-probes           dd/helm-charts (main) keldog-dd
ERROR: Permission to DataDog/helm-charts.git denied to ckelner.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
Just an example of someone else forking: https://github.com/DataDog/helm-charts/pull/1919

Reply:
> I didn't have writes to the repo which is why I created a branch. When I try to push a new branch I get the following

Aha.  Gotcha.  Yeah, we'll have to make it work then 👍